Cargo.toml

```diff
@@ -38,7 +38,7 @@ url = { version = "2.5.2", features = ["serde"] }
 
 sp1-core = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
 sp1-precompiles = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
-sp1-recursion-compiler = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
+sp1-recursion-compiler = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.8-testnet" }
 sp1-recursion-core = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
 sp1-sdk = { git = "https://github.com/succinctlabs/sp1", tag = "v1.0.5-testnet" }
```

Comment:
It's probably better to update all these deps together.

Reply:
We can't for now as the testnet of SP1 was configured for 1.0.5.
Will try to make the upgrade on another PR, will see